registry/test/apps.js

```diff
@@ -0,0 +1,388 @@
+            timeout: 1000,
+            primary: true
+        },
+        assetsDiscoveryUrl: 'http://127.0.0.1:1234/_spa/dev/assets-discovery',
+        // dependencies: {},
+        // props: {},
+        // initProps: {},
+    }),
+    incorrect: Object.freeze({
+        name: 123,
+        spaBundle: 456,
+        cssBundle: 789,
+        ssr: 456,
+        assetsDiscoveryUrl: 789,
+        dependencies: 456,
+        props: 789,
+        initProps: 456,
+    }),
+    updated: Object.freeze({
+        name: '@portal/ncTestAppReactssr',
+        spaBundle: 'http://localhost:1234/ncTestAppReactssrUpdated.js',
+        cssBundle: 'http://127.0.0.1:1234/ncTestAppReactssrUpdated.css',
+        ssr: {
+            src: "http://127.0.0.1:1234/fragmentUpdated",
+            timeout: 2000,
+            primary: false,
+        },
+        assetsDiscoveryUrl: 'http://127.0.0.1:1234/_spa/dev/assets-discoveryUpdated',
+        dependencies: {
+            react: 'https://cdnjs.cloudflare.com/ajax/libs/react/16.8.6/umd/react.development.js',
+            'react-dom': 'https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.8.6/umd/react-dom.development.js',
+        },
+        props: {
+            fragmentModuleName: 'reactssr-app-mainUpdated',
+            assetsPath: 'http://127.0.0.1:3001/uisamplereactUpdated',
+            locationStrategy: 'browserHistoryUpdated',
+        },
+        initProps: {
+            assetsPath: 'http://127.0.0.1:3001/uisamplereact',
+        },
+    }),
+
+    create: async () => {
+        await request.post(example.url).send(example.correct);
+    },
+    delete: async () => {
+        await request.delete(example.url + example.encodedName);
+    },
+};
+example.encodedName = encodeURIComponent(example.correct.name);
+
+describe(`Tests ${example.url}`, () => {
+    before('should work simple "create" and "delete"', async () => {
+        await request.post(example.url).send(example.correct).expect(200)
+        await request.get(example.url + example.encodedName).expect(200);
+        await request.delete(example.url + example.encodedName).expect(204);
+        await request.get(example.url + example.encodedName).expect(404);
+    });
+    afterEach(example.delete);
+    describe('Create', () => {
+        it('should not create record without a required field: name', async () => {
+            const response = await request.post(example.url)
+            .send(_.omit(example.correct, 'name'))
+            .expect(422, '"name" is required');
+
+            expect(response.body).deep.equal({});
+        });
+
+        it('should not create record without a required field: spaBundle', async () => {
+            const response = await request.post(example.url)
+            .send(_.omit(example.correct, 'spaBundle'))
+            .expect(422, '"spaBundle" is required');
+
+            expect(response.body).deep.equal({});
+        });
+
+        it('should not create record with incorrect type of field: name', async () => {
```

Comment:
Well, you wrote good test cases
Actually, I have one notice, it would be good if you have only one expectation each of test case
It is antipattern to have bigger than one expect in one test case
I suggest to separate test case into several
Please check all of your test cases